packages/IMAPClient-Tests.package/ICAddressBookTest.class/instance/testAddEntry.st

```diff
@@ -1,7 +1,7 @@
 as yet unclassified
 testAddEntry
 
-	self addressBook add: self addressBookEntry.
+	self addressBook addEmail: self emailAddress.
 	
-	self assert: self addressBook entries size equals: 1
+	self assert: self addressBook size equals: 1
```

Comment:
Why do you hardcode the size 1 here and keep it variable in the `testChangeEntryToBeDuplicated` method? If this test runs after another insert test it would fail, correct?

Reply:
Either a variable is missing here or it is not needed in `testChangeEntryToBeDuplicated`.

Reply:
Because we expect exactly one element in the list after we added one.
Afaik the tests run independently. The variable in `testChangeEntryToBeDuplicated` simply is a simply more generic approach.